### DISCUSSION TITLE
How to create new stats cards?

### DISCUSSION THREAD

**Discussion by yaten2302** [anuraghazra/github-readme-stats#3183] [Q&A] [answered]
Hi, actually I've a doubt that how can new stats card be created? <br><br>
![image](https://github.com/anuraghazra/github-readme-stats/assets/129659514/91cd2cc5-0584-410d-b605-9fb2c2eb9069)
<br><br>
Like these are some of the stats cards which are already there, but if we want to add some more cards, then how can we do this?
Actually, I was not able to understand that how to create new cards.

Can anyone please help me out with this 🙏

**Comment by qwerty541** [comment, marked as answer]
You can check some my recent pull requests for examples.

Here is adding a new GitHub Gist card: https://github.com/anuraghazra/github-readme-stats/pull/3064

Here is adding a new pie layout for Top Languages card: https://github.com/anuraghazra/github-readme-stats/pull/2709

**Reply by yaten2302** [reply]
Got it, thanks 👍
